src/NzbDrone.Core/ImportLists/AniList/List/AniListParser.cs

```diff
@@ -0,0 +1,143 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using NLog;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Common.Instrumentation;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.ImportLists.Exceptions;
+using NzbDrone.Core.ImportLists.Simkl;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.ImportLists.AniList.List
+{
+    public class AniListParser : IParseImportListResponse
+    {
+        private static readonly Logger Logger = NzbDroneLogger.GetLogger(typeof(SimklParser));
+
+        private readonly Dictionary<int, MediaMapping> _mappings;
+        private readonly AniListSettings _settings;
+
+        public AniListParser(AniListSettings settings, Dictionary<int, MediaMapping> mappings)
+        {
+            _mappings = mappings;
+            _settings = settings;
+        }
+
+        public virtual IList<ImportListItemInfo> ParseResponse(ImportListResponse importListResponse)
+        {
+            return ParseResponse(importListResponse, out _);
+        }
+
+        public IList<ImportListItemInfo> ParseResponse(ImportListResponse importListResponse, out PageInfo pageInfo)
+        {
+            var result = new List<ImportListItemInfo>();
+
+            if (!PreProcess(importListResponse))
+            {
+                pageInfo = null;
+                return result;
+            }
+
+            var jsonResponse = STJson.Deserialize<MediaPageResponse>(importListResponse.Content);
+
+            if (jsonResponse?.Data?.Page?.MediaList == null)
+            {
+                pageInfo = null;
+                return result;
+            }
+
+            // Anilist currently does not support filtering this at the query level, they will get filtered out here.
+            var filtered = jsonResponse.Data.Page.MediaList
+                .Where(x => ValidateMediaStatus(x.Media));
+
+            foreach (var item in filtered)
+            {
+                var media = item.Media;
+
+                if (_mappings.TryGetValue(media.Id, out var mapping))
+                {
+                    // Base required data
+                    var entry = new ImportListItemInfo()
+                    {
+                        TvdbId = mapping.Tvdb.Value,
+                        Title = media.Title.UserPreferred ?? media.Title.UserRomaji ?? default
```

Comment:
`?? default` isn't required since it's null for a string

```suggestion
                        Title = media.Title.UserPreferred ?? media.Title.UserRomaji
```